docs/writing-tests/testharness-tutorial.md

```diff
@@ -0,0 +1,390 @@
+# testharness.js tutorial
+
+<!--
+Note to maintainers:
+
+This tutorial is designed to be an authentic depiction of the WPT contribution
+experience. It is not intended to be comprehensive; its scope is intentionally
+limited in order to demonstrate authoring a complete test without overwhelming
+the reader with features. This should be updated as typical WPT usage patterns
+change, but please weigh extensions against the demotivating effect that a
+lengthy guide can have on new contributors.
+-->
+
+Let's say you've discovered that WPT doesn't have any tests for how [the Fetch
+API](https://fetch.spec.whatwg.org/) sets cookies from an HTTP response. This
```

Comment:
I agree, but I also think covering that topic in sufficient depth will significantly expand the scope of this tutorial (especially given the need for a more complicated motivating example). Do you think it would work better as a standalone tutorial?

Reply:
I think that could work, yes. I don't think it needs to be in the same tutorial, or available immediately. A nice add for the future, perhaps?